spec/views/pages/index.html.erb_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe "pages/index.html.erb", type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You should feel safe deleting this file as well.

Reply:
done